### PR TITLE
OS Packages - add admon & update client matrices

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/11448

### CODE REVIEW

File: modules/client-configuration/pages/supported-features-es.adoc [hunk 1]
```diff
@@ -18,7 +24,7 @@ The icons in this table indicate:
 |===
 | Feature | {slsa}{nbsp}ES{nbsp}6 |  {slsa}{nbsp}ES{nbsp}7 |  {slsa}{nbsp}ES{nbsp}8
 | Client | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| Operating system packages | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| {suse} system packages | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
```
Comment (paususe) [inline modules/client-configuration/pages/supported-features-es.adoc:27]:
```suggestion
| System packages | {suse} | {suse} | {suse}
```

File: modules/client-configuration/pages/supported-features-rh.adoc [hunk 2]
```diff
@@ -18,7 +25,7 @@ The icons in this table indicate:
 |===
 | Feature | {rhela}{nbsp}6 | {rhela}{nbsp}7 | {rhela}{nbsp}8
 | Client | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| Operating system packages | icon:times[role="danger"] | icon:times[role="danger"] | icon:times[role="danger"]
+| {redhat} system packages | icon:times[role="danger"] | icon:times[role="danger"] | icon:times[role="danger"]
```
Comment (paususe) [inline modules/client-configuration/pages/supported-features-rh.adoc:28]:
```suggestion
| System packages | {redhat} | {redhat} | {redhat}
```

File: modules/client-configuration/pages/supported-features-ubuntu.adoc [hunk 3]
```diff
@@ -25,7 +31,7 @@ The icons in this table indicate:
 |===
 | Feature | {ubuntu}{nbsp}16.04 | {ubuntu}{nbsp}18.04
 | Client | icon:check[role="green"] | icon:check[role="green"]
-| Operating system packages | icon:times[role="danger"] | icon:times[role="danger"]
+| Canonical system packages | icon:times[role="danger"] | icon:times[role="danger"]
```
Comment (paususe) [inline modules/client-configuration/pages/supported-features-ubuntu.adoc:34]:
```suggestion
| System packages | Canonical | Canonical
```

File: modules/client-configuration/pages/supported-features-centos.adoc [hunk 4]
```diff
@@ -25,7 +27,7 @@ The icons in this table indicate:
 |===
 | Feature | {centos}{nbsp}6 | {centos}{nbsp}7 | {centos}{nbsp}8
 | Client | icon:question[role="gray"] (plain CentOS) / icon:check[role="green"] (with {es}) | icon:question[role="gray"] (plain CentOS) / icon:check[role="green"] (with {es}) | icon:question[role="gray"] (plain CentOS) / ``Salt`` (with {es})
-| Operating system packages  |  icon:times[role="danger"] (plain CentOS) / icon:check[role="green"] (with {es}) | icon:times[role="danger"] (plain CentOS)/icon:check[role="green"] (with {es}) | icon:times[role="danger"] (plain CentOS)/``Salt`` (with {es})
+| {centos} system packages |  icon:times[role="danger"] (plain CentOS) / icon:check[role="green"] (with {es}) | icon:times[role="danger"] (plain CentOS)/icon:check[role="green"] (with {es}) | icon:times[role="danger"] (plain CentOS)/``Salt`` (with {es})
```
Comment (paususe) [inline modules/client-configuration/pages/supported-features-centos.adoc:30]:
```suggestion
| System packages |  {centos} Community / {suse} (with {es}) | {centos} Community / {suse} (with {es}) | {centos} Community / {suse} (with {es})
```

File: modules/client-configuration/pages/supported-features-debian.adoc [hunk 5]
```diff
@@ -25,7 +31,7 @@ The icons in this table indicate:
 |===
 | Feature | {debian}{nbsp}9 | {debian}{nbsp}10
 | Client | icon:check[role="green"] | icon:check[role="green"]
-| Operating system packages | icon:times[role="danger"] | icon:times[role="danger"]
+| {debian} system packages | icon:times[role="danger"] | icon:times[role="danger"]
```
Comment (paususe) [inline modules/client-configuration/pages/supported-features-debian.adoc:34]:
```suggestion
| System packages | {debian} Community | {debian} Community
```

File: modules/client-configuration/pages/supported-features-sles.adoc [hunk 6]
```diff
@@ -19,7 +26,7 @@ The icons in this table indicate:
 |===
 | Feature | {sle}{nbsp}11 | {sle}{nbsp}12 | {sle}{nbsp}15 | {opensuse}{nbsp}15.1
 | Client | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"]
-| Operating system packages  | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"]
+| {suse} system packages  | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"]
```
Comment (paususe) [inline modules/client-configuration/pages/supported-features-sles.adoc:29]:
```suggestion
| System packages  | {suse} | {suse} | {suse} | {opensuse} Community
```

**Comment by keichwa** [comment]
On 2020-05-18 11:58, Pau Garcia Quiles wrote:

> Do we need the "trademark" symbol next to Red Hat, Canonical and
> Debian? What are the rules for that?

Usually, such a general statement is sufficient (quoted from the SLES 15 
SP1
Deployment Guide, single-html, near the copyright statement (back 
title),
https://documentation.suse.com/sles/15-SP1/single-html/SLES-deployment/#book-sle-deployment):

>  For SUSE trademarks, see https://www.suse.com/company/legal/. All 
> other third-party trademarks are the property of their respective 
> owners. Trademark symbols (®, ™ etc.) denote trademarks of SUSE and its 
> affiliates. Asterisks (*) denote third-party trademarks.

I think only it in doubt and at the first occurrence we add a (pseudo) 
trademark
symbols.  For up to date info, mail Ciaran Farrell.

-- 
Karl Eichwalder                    SUSE Software Solutions Germany GmbH
                                    Maxfeldstraße 5
                                    90409 Nürnberg, Germany
Geschäftsführer: Felix Imendörffer (HRB 36809, AG Nürnberg)

**Comment by Loquacity** [comment]
> Having the reds ("danger" icon) looks like something is unsupported by us, i. e. missing features, while what it actually means in this row is "support does not come from SUSE".
> 
> Do we need the "trademark" symbol next to Red Hat, Canonical and Debian? What are the rules for that?

Not usually in technical documentation, that kind of thing should be covered in our legal statement. I haven't seen anything specific on that at SUSE, but can bring it up at the docs meeting tonight if you like.

**Pull request merged**